src/geom.hpp

```diff
@@ -84,6 +84,23 @@ class point_t
         return !(a == b);
     }
 
+    /// Give points an (arbitrary) order.
+    [[nodiscard]] constexpr friend bool operator<(point_t a, point_t b) noexcept
+    {
+        if (a.x() == b.x()) {
+            return a.y() < b.y();
```

Comment:
This branch is not tested - the test coverage has two points with equal y, but not ones with equal x.

Reply:
I have amended the test.